### ISSUE TITLE
[lens] authinfo2 file for S3QL

### ISSUE THREAD

**Issue by Salokyn** [hercules-team/augeas#646]
S3QL is a file system that stores all its data online using storage services like Google Storage, Amazon S3, or OpenStack. S3QL effectively provides a hard disk of dynamic, infinite capacity that can be accessed from any computer with internet access.

More info at: http://www.rath.org/s3ql-docs/about.html

This tool read configuration from files that are stored by default in $HOME/.s3ql/authinfo2

Description of this file at: http://www.rath.org/s3ql-docs/authinfo.html

Could someone write a lens to handle these authinfo2 files please ?

**Comment by raphink** [comment]
This looks like a form of INI file, which shouldn't be too hard to support. I'll welcome a PR if you write it.

**Comment by Salokyn** [comment]
I don't know how to do write lenses, but I'll try.

**Comment by Salokyn** [comment]
Is it possible to specify a path relative to HOME in `incl`?

Something like:
```
let filter = (incl "$HOME/.s3ql/authinfo2")
```

**Comment by raphink** [comment]
Yes, see for ex https://github.com/hercules-team/augeas/blob/master/lenses/dput.aug#L65
